createcaslibjson.py

```diff
@@ -45,14 +45,6 @@
 import sys
 from sharedfunctions import callrestapi, getfolderid, file_accessible, printresult, getapplicationproperties
```

Comment:
Should the getclicommand definition be included here?

Reply:
And missing the lines that'd call it too?

```
# get cli location from properties, check that cli is there if not ERROR and stop
clicommand=getclicommand()
```

Reply:
That one doesn't actually use the CLI. I only added it to tools that actually make a CLI call